core/variables.js

```diff
@@ -500,10 +500,11 @@ Blockly.Variables.getOrCreateVariablePackage = function(workspace, id, opt_name,
  */
```

Comment:
Since the return is explicitly null-able, I think should be changed to {?Blockly.VariableModel} (although I know its not a new thing)

Reply:
``Blockly.VariableModel`` is the same as ``?Blockly.VariableModel``, so there's no need to add that